.github/ISSUE_TEMPLATE/----source-request.yml

```diff
@@ -0,0 +1,22 @@
+name: ➕ Source Request
+description: Request to add a new source
+labels: ['➕ source request']
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the request as much as possible so we can efficiently process your request.
```

Comment:
Hope you don't mind. I just made a small grammar update. 👍

Reply:
Thank you for your help!